### ISSUE TITLE
current version of yarn (v1.22.21) gives error instead of installing medusa-cli

### ISSUE THREAD

**Issue by MitchelFrick** [medusajs/medusa#6385]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

When installing yarn with corepack (per most recent instructions on [yarnpkg.com](https://yarnpkg.com/getting-started/install) linked in medusa documentation) receive the following error when setting up medusa-cli:

```
medusa@medusa-backend:~$ yarn global add @medusajs/medusa-cli
Usage Error: The 'yarn global' commands have been removed in 2.x - consider using 'yarn dlx' or a third-party plugin instead

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] [--require #0] <scriptName> ...
```

After installing yarn with 
`sudo apt install yarn`
medusa-cli install completes normally

corepack version of yarn is v1.22.21
apt install version of yarn is v1.22.19

### How can this issue be resolved?

1. Change documentation to specify: sudo apt install yarn
2. Alternatively update medusa-cli install command to comply with new requirements introduced in yarn v1.22.21 

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

**Comment by xyzones** [comment]
https://github.com/medusajs/medusa/pull/6186 it will get to the docs at some point

**Comment by shahednasser** [comment]
Closing this since it's fixed by #6186
